docs/troubleshooting/management/ts-management.asciidoc

```diff
@@ -33,7 +33,23 @@ TIP: If the {endpoint-cloud-sec} integration policy is not the cause of the `Unh
 .Disabled to avoid potential system deadlock (Linux)
 [%collapsible]
 ====
-This section is a placeholder for future documentation.
+If you have an `Unhealthy` {agent} status with the message `Disabled due to potential system deadlock`, that means malware protection was disabled on the {endpoint-cloud-sec} integration policy due to errors while monitoring a Linux host.
+
+You can resolve the issue by configuring the policy's <<linux-file-monitoring,advanced settings>> related to *fanotify*, a Linux feature that monitors file system events. By default, {endpoint-cloud-sec} works with fanotify to monitor specific file system types that Elastic has tested for compatibility, and ignores other unknown file system types.
+
+If your network includes nonstandard, proprietary, or otherwise unrecognized Linux file systems that cause errors while being monitored, you can configure {endpoint-cloud-sec} to ignore those file systems. This allows {endpoint-cloud-sec} to resume monitoring and protecting the hosts on the integration policy.
+
+CAUTION: Ignoring file systems can create gaps in your security coverage. Use additional security layers for any file systems ignored by {endpoint-cloud-sec}.
+
+. Go to *Manage* -> *Policies*, then click a policy's name.
```

Comment:
The only suggestion I have for this section is to add a lead-in sentence, or something to briefly describe what these steps help you do (i.e., the goal of this task). Perhaps something like:
To configuring your policy’s advanced settings related to fanotify, do the following:

Or, something like:
Follow the steps below to configure Endpoint and Cloud Security to ignore specified file systems. Once you save the policy, malware protection is re-enabled.

Reply:
One more comment - I took a look at this again and I think the whole section functions as a lead-in to the steps. Adding an introductory sentence would offer an extra layer of clarity, but it could also be a bit excessive for such a short section. Either way, feel free to reject my initial suggestion if it adds unnecessary "fluff" to the troubleshooting steps. I definitely don't want to advocate replicating content where it's not needed. :)

Reply:
@nastasha-solomon Yeah I don't think lead-ins are always necessary just for their own sake, but a brief one here wouldn't hurt, just to shift the focus and orient the reader.